setup.cfg

```diff
@@ -19,3 +19,14 @@ max-line-length = 90
 #  W503 requires that binary operators be at the end, not start, of lines. Erik doesn't like it.
 #  E203 is contrary to PEP8.
 ignore = W503,E203
+
+[isort]
+line_length = 89
+not_skip = __init__.py
+sections=FUTURE,STDLIB,COMPAT,THIRDPARTY,TWISTED,FIRSTPARTY,LOCALFOLDER
+default_section=THIRDPARTY
+known_first_party = synapse
```

Comment:
do we need to include `tests` in this too?

Reply:
Good catch.